src/main/java/org/digitalmediaserver/nsis/MakeMojo.java

```diff
@@ -293,13 +293,26 @@ private void validate() throws MojoExecutionException {
 			Path path = Paths.get(makensisExecutable);
 			makensisExecutable = path.toRealPath().toString();
 			if (autoNsisDir && isBlank(nsisDir)) {
+				if (Files.isSymbolicLink(path)) {
```

Comment:
Maybe we could simply do as I've suggested above? I can't see a reason to call `toRealPath()` multiple times.